_posts/2020-05-15-model-inversion-attacks/model_inversion_differential_privacy.Rmd

```diff
@@ -0,0 +1,740 @@
+---
+title: "Hacking deep learning: model inversion attack by example"
+description: > 
+  Compared to other applications, deep learning models might not seem too likely as victims of privacy attacks. However, methods exist to determine whether an entity was used in the training set (an adversarial attack called member inference), and techniques subsumed under "model inversion" allow to spy on raw data input given just model output (and sometimes, context information). This post shows an end-to-end example of model inversion, and explores mitigation strategies using TensorFlow Privacy. 
```

Comment:
> allow to spy on raw data input given just model output (and sometimes, context information)

I think you need a word between "allow" and "to spy" (allow _what_ to spy?)

Reply:
hm... or another verb...? reconstruct, I think...